runtime-common/stdlib/string/string-context.h

```diff
@@ -45,6 +45,9 @@ class StringLibContext final : vk::not_copyable {
   // mb_string context
   bool detect_incorrect_encoding_names{};
 
+  // from-json-processor context
+  string last_json_processor_error;
```

Comment:
I think we need separate context for (de)serialization routines. I'd like to prevent our contexts from being a trash for former global variables. Let's have something like `SerializationContext`. Also, let's create a new directory `stdlib/serialize/`

Reply:
After adding a new context to `runtime-common`, please refer to how string contexts are included in k2 runtime

Reply:
+, introduce serparate SerializationContext in runtime-common/serialize/serialization-context.h